File: main6.py
import math
print("Даньков Артем,27 В, ІС-01")
i = True
while i:
    try:
        a = float(input("Введіть перший символ а:"))
        b = float(input("Введіть другий символ b:"))
        i = 0
    except:
        continue


def u(a, b):
    if a < b:
        return a
    else:
        return b


def v(a, b):
    if a*b < a+b:
        return a*b
    else:
        return a+b


def w(a, b):
    c = u(a, b)+pow(v(a, b), 2)
    if c < math.pi:
        return c
    else:
        return math.pi


print("U= "+str(u(a, b)))
print("V= "+str(v(a, b)))
print("W= "+str(w(a, b)))

File: main6.cpp
#include <iostream>
#define _USE_MATH_DEFINES
#include <math.h>


using namespace std;


double U(double a,double b)
{
if(a<b) return a;
else return b;
}

double V(double a,double b)
{
if(a*b<a+b) return a*b;
else return a+b;
}

double W(double a,double b)
{
double c = U(a,b)+V(a,b)*V(a,b);
if (c<M_PI) cout<<c;
else cout<<M_PI;
}

int main() {
    double a,b;
    cout<<"Put first digit a: ";
    cin>>a;
    cout<<"Put second digit b: ";
    cin>>b;
    W(a,b);
    return 0;
    }
#include <iostream>
#define _USE_MATH_DEFINES
#include <math.h>


using namespace std;




int main() {
    cout<<"Dankov Artem,IS-01,27-V";
    double a,b;
    double U(double a,double b);
    double V(double a,double b);
    double W(double a, double b);
    cout<<"Put first digit a: ";
    cin>>a;
    cout<<"Put second digit b: ";
    cin>>b;
    W(a,b);
    return 0;
}
double U(double a,double b)
{
    double min=0;
    if(a<b) {
        min=a;
        return min;
    }
    else {
        min=b;
        return min;
        cout<<"U="<<b<<endl;
    }
}

double V(double a,double b)
{
    double min=0;
    if(a*b<a+b) {
        min=a*b;
        return a*b;
        cout<<"V="<<a*b<<endl;
    }
    else {
        min=a+b;
        return a+b;
        cout<<"V="<<a+b<<endl;
    }
}

void W(double a,double b)
{
    cout<<"U="<<U(a,b)<<endl;
    cout<<"V="<<V(a,b)<<endl;
    double min;
    double c = U(a,b)+V(a,b)*V(a,b);
    if (c<M_PI) {
        min=c;
        cout<<"W="<<c;
    }
    else {
        min=M_PI;
        cout<<"W="<<M_PI;
    }
}
